pkg/util/machine/topology.go

```diff
@@ -513,8 +513,12 @@ type NumaDistanceInfo struct {
 }
 
 type ExtraTopologyInfo struct {
-	NumaDistanceMap              map[int][]NumaDistanceInfo
-	SiblingNumaMap               map[int]sets.Int
+	NumaDistanceMap map[int][]NumaDistanceInfo
+	SiblingNumaMap  map[int]sets.Int
+
+	// SiblingNumaMBWAllocatableMap and SiblingNumaMBWCapacityMap are
```

Comment:
this comment is too confusing to understand ... do you just want to say `SiblingNumaAvgMBWAllocatableMap` and `SiblingNumaAvgMBWCapacityMap`?